Hunk 1: Fix dectection of fox libs.
Hunk 2,3: Remove leftover file so temp directory is cleaned.
Hunk 3: Use pkgconf for QT_LIBS. The other method produces the full path to the
library which libtool seems to discard.

--- configure.orig	2024-12-17 18:57:54 UTC
+++ configure
@@ -23442,7 +23442,7 @@ if test "${with_fox_libs+set}" = set; then :
 
 # Check whether --with-fox-libs was given.
 if test "${with_fox_libs+set}" = set; then :
-  withval=$with_fox_libs; fox_libs="$withval"
+  withval=$with_fox_libs; local_fox_libs="$withval"
 else
 
       ac_ext=cpp
@@ -25024,6 +25024,7 @@ $as_echo_n "checking for Qt... " >&6; }
     am_have_qt_dir=`mktemp -d`
     am_have_qt_pro="$am_have_qt_dir/test.pro"
     am_have_qt_makefile="$am_have_qt_dir/Makefile"
+    am_have_qt_stash="$am_have_qt_dir/.qmake.stash"
     # http://qt-project.org/doc/qt-5/qmake-variable-reference.html#qt
     cat > $am_have_qt_pro << EOF
 win32 {
@@ -25039,8 +25040,8 @@ EOF
 EOF
     $QT_QMAKE $am_have_qt_pro -o $am_have_qt_makefile
     QT_CXXFLAGS=`cd $am_have_qt_dir; make -s -f $am_have_qt_makefile CXXFLAGS INCPATH`
-    QT_LIBS=`cd $am_have_qt_dir; make -s -f $am_have_qt_makefile LIBS`
-    rm $am_have_qt_pro $am_have_qt_makefile
+    QT_LIBS=`pkgconf --libs Qt5Widgets gl`
+    rm $am_have_qt_pro $am_have_qt_makefile $am_have_qt_stash
     rmdir $am_have_qt_dir
 
     # Look for specific tools in $PATH
